live-build/misc/ansible-roles/appliance-build.qa-internal/tasks/main.yml

```diff
@@ -50,6 +50,7 @@
       - libpam0g-dev
       - libssl-dev
       - python-pip
+      - snmptrapd
```

Comment:
While this will work, I think it would be more fitting to add this to the list above instead. You may also want to add a quick comment mentioning why we need this.

Reply:
Just to confirm this should be added above `ant` entry?

Reply:
No, right below `nftables` on line 21, which was also added by QA and is needed specifically for QA workflows.

Reply:
got it. should have read the comments. thanks.

Reply:
Can you remove `snmptrapd` from this list?

Reply:
Yea. I was figuring out a commit issue i was seeing while pushing to the fork. I have the updated code. Please check